scripts/qa_crowbarsetup.sh

```diff
@@ -4442,11 +4527,11 @@ function ping_fips
 # Use $heat_stack_params to provide parameters to heat template
 function oncontroller_testpreupgrade
 {
-    create_cmd="heat --insecure stack-create upgrade_test -f "
-    list_cmd="heat --insecure stack-list"
+    create_cmd="heat stack-create upgrade_test -f "
+    list_cmd="heat stack-list"
```

Comment:
this change needs to be reverted (as it is only for cloud6 which won't get the fixes)